nbs/03_matcher.ipynb

```diff
@@ -345,6 +345,13 @@
     "    # Then filter the list, to contain only the filtered match whose unfiltered version matches the condition\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #35.        #TODO: call add_collections fore each match in _remove_duplicated_matches(filtered_matches) and yield from the result # Collections Feature
</pre><p>the two phase matching logic should be in this nb.</p><p> </p><p>Try to see if you can make 2 phase matching a reduction of 1 phase matching with constant nodes.</p><p> </p><p>It will make the implementation much easier.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/graph_rewrite/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>On it, and we'll discuss this with you in our meeting</p>